### PR TITLE
Update 03_Web_Browser_Differences.md

### PR DESCRIPTION
**Pull request by DorsetDigital**
Update content

The current page is quite opinionated and offers no citation for the assertions about speed or security.   
It also references Internet Explorer and not 'Edge' which is currently shipped with Windows 10.

I've tried to provide a more generic approach to the recommendations.

### CODE REVIEW

File: docs/en/02_Creating_pages_and_content/03_Web_Browser_Differences.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ summary: Guidance on the best web browsers to use when editing content in Silver
 
 # Web browser differences
 
-It's highly recommended to use either [Google Chrome](http://google.com/chrome) or [Mozilla FireFox](http://www.mozilla.org/en-GB/firefox/) as your web browser, as Internet Explorer doesn't support some features such as inserting multiple images at once, or resizing images whilst maintaining their proportions.
+Due to the variety of operating systems and web browsers available, it can sometimes be difficult to know which to use.  The SilverStripe CMS is tested on a number of platforms to ensure maximum compatibility and generally speaking, most modern browsers will be suitable for working with SilverStripe.
```
Comment (dhensby) [inline docs/en/02_Creating_pages_and_content/03_Web_Browser_Differences.md]:
```diff
-and generally speaking,
+and, generally speaking,
```

Perhaps "all modern standards compliant browsers"?

**Comment by dhensby** [comment]
Yeah, you're right, and this is pretty arbitrary, SS will no doubt work just as well with Opera as it does with Chrome.

I'm not even sure we need to make the Chrome / FireFox recommendation unless we want to say something like "we focus primarily on supporting Chrome and FireFox so these are the safest browsers to use"?

**Comment by DorsetDigital** [comment]
I've made the text change.  As for the inclusion of the specific recommendation, I guess that's a policy decision for `<insert responsible people>` to make.

**Comment by dhensby** [comment]
OK - let's get this in either way, the phrasing change I'm suggesting doesn't really alter the sentiment

**Comment by sachajudd** [comment]
Thanks @DorsetDigital! 🙂

**Pull request merged**